src/17D6A0.c

```diff
@@ -23,47 +23,50 @@ extern ActorBlueprint battle_partner_lakilester;
 extern ActorBlueprint battle_partner_bow;
 extern ActorBlueprint battle_partner_twink;
 
-s16 D_80280FC0[] = {
-    10, 90, 0, 50,
-    3, 11, 0, 50,
-    1, 45, 0, 0,
-    0, 0, 0, 0,
-    0, 0, 0, 0,
-    0, 0, 0, 0,
-    0, 0, 0, 0,
-    0, 0, 0, 0,
-    0, 0, 0, 50,
-    1, 49, 0, 50,
-    1, 49, 0, 0,
-    0, 0, 0, 0,
-    0, 0, 0, 0,
-    0, 0, 0, 0,
-    0, 0, 0, 0,
-    0, 0, 0, 0,
-    0, 0, 0, 50,
-    1, 50, 0, 50,
-    1, 50, 0, 0,
-    0, 0, 0, 0,
-    0, 0, 0, 0,
-    0, 0, 0, 0,
-    0, 0, 0, 0,
-    0, 0, 0, 0,
-    0, 0, 0, 50,
-    3, 10, 0, 50,
-    3, 10, 0, 0,
-    0, 0, 0, 0,
-    0, 0, 0, 0,
-    0, 0, 0, 0,
-    0, 0, 0, 0,
-    0, 0, 0, 0,
-    0, 0, 0, 80,
-    3, 10, 0, 20,
-    3, 11, 0, 0,
-    0, 0, 0, 0,
-    0, 0, 0, 0,
-    0, 0, 0, 0,
-    0, 0, 0, 0,
-    0, 0, 0, 0, 0, 0,
+UnkPartnerThing D_80280FC0 = {
+    10, 90,
+    {
+        50, 196619,
```

Comment:
The second field here are player anim IDs.
e.g., 196619 = 0x1002D = ANIM_Mario_1002D

Reply:
ah nice - I thought that may be the case, but I wasn't sure. I'll add those